CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# v1.20.5 (2024-07-30)
+
+## OS Changes
+
+* Update docker-engine to v25.0.6 ([#4111])
+* Update containerd to 1.6.34 ([#4113])
```

Comment:
nit: For older BR versions in the Changelog, I see that docker/containerd updates are in variety of sections: ECS, Build Changes, Security Fixes, etc.

Do we need to be consistent in this area, or is it a non-issue?

Reply:
I'm following precedent of our two most recent `containerd` version bumps:

* Bottlerocket 1.19.4 https://github.com/bottlerocket-os/bottlerocket/blob/develop/CHANGELOG.md#os-changes-6
* Bottlerocket 1.19.0 https://github.com/bottlerocket-os/bottlerocket/blob/develop/CHANGELOG.md#os-changes-10

and as for the `docker-engine` update, most recently we listed this in OS changes: https://github.com/bottlerocket-os/bottlerocket/blob/develop/CHANGELOG.md#os-changes. It's semantics, IMO: `docker-engine` itself is not a container orchestrator in the sense that the `ecs-agent` is; on the other hand, yes, `docker-engine` is a direct dependency of the `ecs-agent` orchestrator. I'm going to opt to leave as is.